README.md

```diff
@@ -0,0 +1,3 @@
+# code_review
+
+You ask what I want to do with it. I don't know yet, but I am sure I will know before I finish writing it.
```

Comment:
README should contain at minimum brief description of what the script does and why.

Reply:
Yup!